Utils/gitlab_triggers/trigger_upload_flow_build.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
```

Comment:
Nice bash script! Any reason you want to do this as opposed to a python script? It would probably be a lot shorter...

Reply:
since this will run locally on the developers pc, I don't want it to be python version/libraries dependent. That said I don't feel that strongly about it, if you think python is more maintainable i'll rewrite.

Reply:
This is an existing bash script, not sure re-writing is needed. @DinaMeylakh any changes here rather than the request to gitlab instead of circleci?

Reply:
I updated the existing one as well as created a new one so we can safely delete the circle one once it is not needed.